network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -390,17 +393,39 @@ class ProductPage(FoundationMetadataPageMixin, Page):
         ),
     ]
 
+    @property
+    def product_type(self):
+        if isinstance(self, SoftwareProductPage):
+            return "software"
+        elif isinstance(self, GeneralProductPage):
+            return "general"
+        else:
+            return "unknown"
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        context['product'] = self
+        context['categories'] = BuyersGuideProductCategory.objects.all()
+        context['mediaUrl'] = settings.CLOUDINARY_URL if settings.USE_CLOUDINARY else settings.MEDIA_URL
+        context['coralTalkServerUrl'] = settings.CORAL_TALK_SERVER_URL
+        context['pageTitle'] = pgettext(
+                'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+                '*privacy not included') + f' - {self.title}'
```

Comment:
Just a nit, if we're using an fstring anyway:

```suggestion
        context['pageTitle'] = f'{pgettext(
          'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
          '*privacy not included'
        )} - {self.title}'
```